chore(README.md): fix formatting and add OpenVSX badge (#458)
coder/vscode-coder: README.md

- Works in air-gapped or restricted networks. Just connect to your Coder
  deployment!
- Supports multiple editors: VS Code, Cursor, and Windsurf.

> [!NOTE]

> > The extension builds on VSCode provided implementations of SSH. Make

Reviewer: ```suggestion
> The extension builds on VS Code-provided implementations of SSH. Make
```

Author: I see a mix of usage of VSCode and VS Code. Can we set a linter rule to enforce the correct `VS Code`?

> > sure you have the correct ssh extension installed for your editor

Reviewer: ```suggestion
> sure you have the correct SSH extension installed for your editor
```

> > (ms-vscode-remote.remote-ssh or codeium.windsurf-remote-openssh for windsurf)

Reviewer: ```suggestion
> (`ms-vscode-remote.remote-ssh` or `codeium.windsurf-remote-openssh` for Windsurf).
```
